rootfs/network/parse-config

```diff
@@ -141,12 +141,6 @@ server 2.pool.ntp.org
 server 3.pool.ntp.org
 EOF
     fi
-
-    cat <<EOF
-driftfile /tmp/ntp.drift
-pidfile /var/run/ntpd.pid
-logconfig -syncevents -sysevents
```

Comment:
What does deleting these options mean?

Reply:
it means we don't have some extra information files created by ntpd,I don't think anyone cared about the information anyways